README.md

```diff
@@ -1,2 +1,19 @@
 # hw01
 Repository for first CFSS homework assignment
+
+Name:Archer Pan  
+File:HW1  
+Due Date:Jan 11th    
+## **Archer Pan's Brief Biography**
+###**Education**
+I'm currently a MAPSS student concentrating on *Behavioral Science*.I'm researching false memeory and foreign language effect under supervision of Dr.Boaz Keysar.  
+Here is the link to the lab (https://mdl.uchicago.edu/directory/boaz-keysar)  
+Before coming to Uchicago, I received my B.A from University of Illinois at Urbana-Champaign.
+####**Pet**
+I'm a dog lover, I have a five-month-old mininature goldendoodle.  
+Here is a pircture of him:
+http://ww4.sinaimg.cn/mw690/7f6853e5gw1fbfm4ziuj0j20qo0zkn2z.jpg
```

Comment:
What you want to do here might be to embed the image into the markdown file as pointed out by the other peer reviewer